spec/requests/servers_spec.rb

```diff
@@ -1,103 +0,0 @@
-RSpec.describe "Servers" do
```

Comment:
Both of these spec files were erroneously backported, they shouldn't exist in Fine at all.